src/Microsoft.AspNet.Identity.EntityFramework/IdentityServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Identity;
+using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.Framework.ConfigurationModel;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    public static class IdentityServiceCollectionExtensions
+    {
+        public static IdentityBuilder AddIdentity(this IServiceCollection services)
```

Comment:
Interesting that these overload now moves here because the typed moved and hence they are tied to EF. I think it would probably better to get rid of them.